src/truncated/normal.jl

```diff
@@ -141,10 +141,14 @@ function rand(rng::AbstractRNG, d::Truncated{Normal{T},Continuous}) where T <: R
     d0 = d.untruncated
     μ = mean(d0)
     σ = std(d0)
-    a = (d.lower - μ) / σ
-    b = (d.upper - μ) / σ
-    z = randnt(rng, a, b, d.tp)
-    return μ + σ * z
+    if isfinite(μ)
+        a = (d.lower - μ) / σ
```

Comment:
if isfinite()
...
return ..
end
return end

no need for an else with early return

Reply:
no that's alright by me, not worth bothering too much, the main concern is keeping the code homogeneous